lib/rbtrace/cli.rb

```diff
@@ -407,6 +415,10 @@ def self.run
           tracer.puts "=> #{res}"
         end
 
+      elsif opts[:interactive_given]
+        require "rbtrace/interactive/#{opts[:interactive]}"
+        load `which #{opts[:interactive]}`.chomp
```

Comment:
If irb doesn't work then I don't think it should be possible to use it. I don't think it makes sense to use Pry or any other fancy repl either, because there is no true access to the underlying ruby environment and so features like tab completion will never work.

This makes me wonder, should rbtrace implement its own repl loop instead of relying on rirb? It can be a simple `while input=gets` loop, no? What other features does rirb provide that are useful here?